### PR TITLE
Add Basic SKU and KeyVaultId

### PR DESCRIPTION
**Pull request by solankisamir**
- Basic SKU
- KeyVaultId
- MSI Identity support

### CODE REVIEW

**Comment by solankisamir** [comment]
@Tiano2017 can you take a look?

File: schemas/2017-03-01/Microsoft.ApiManagement.json [hunk 1]
```diff
@@ -67,6 +67,17 @@
           ],
           "description": "SKU properties of the API Management service."
         },
+        "identity": {
```
Comment (TianoMS) [inline schemas/2017-03-01/Microsoft.ApiManagement.json]: >identity [](start = 9, length = 8)

do you want to update the tests in /tests/2017-03-01/ to include these changes?

Comment (solankisamir) [reply schemas/2017-03-01/Microsoft.ApiManagement.json]: @TianoMS  added test

**Comment by solankisamir** [comment]
@TianoMS  added tests

**Pull request merged**